report/finished/chapters/choice_PD.tex

```diff
@@ -0,0 +1,14 @@
+\section{Line Follower}
+The robot will use a line follower controller in order to navigate the factory floor. This in combination with \textbf{QR} codes and a path-planning algorithm will give a huge variety in places that the robot can be instructed to go, as long as it's on the grid and denoted with a \textbf{QR} code.
```

Comment:
I tjhink it sould be in other time domain (?). We did all of this, we're not planning to do all of this. For example the robot UTILIZES a line following controller, not WILL use it. Otherwise very good!